bugbug/tools/code_review.py

```diff
@@ -1116,15 +1116,24 @@ def find_similar_patch_comments(self, patch: Patch, limit: int):
 
         patch_set = PatchSet.from_string(patch.raw_diff)
 
-        yielded = 0
+        max_score_per_found = {}  # to avoid repetition of same found
```

Comment:
I expanded the comment to make it clearer for future us :)
```suggestion
        # We want to avoid returning the same comment multiple times. Thus, if
        # a comment matches multiple hunks, we will only consider it once.
        max_score_per_found = {}
```